covidfaq/config.py

```diff
@@ -6,6 +6,8 @@
 class Config(BaseSettings):
     elastic_search_host: str = "faq-master"
     elastic_search_port: int = 9200
+    elastic_search_use_ssl: bool = True
+    elastic_search_verify_certs: bool = True
```

Comment:
I'm pretty sure this is not required if the previous is set to False. To be tested